frontend/www/js/omegaup/components/arena/ContestPractice.test.ts

```diff
@@ -9,7 +9,7 @@ import arena_ContestPractice from './ContestPractice.vue';
 describe('ContestPractice.vue', () => {
   const date = new Date();
 
-  const contestDetails = {
+  const contest = {
```

Comment:
```suggestion
  const contest: types.ContestPublicDetails = {
```

(quitando el tipo de L32).

Reply:
Listo.